datahub-web-react/src/app/entity/chart/ChartEntity.tsx

```diff
@@ -80,6 +79,14 @@ export class ChartEntity implements Entity<Chart> {
                 component: ChartStatsSummarySubHeader,
             }}
             tabs={[
+                {
+                    name: 'Fields',
```

Comment:
Should we call this "Explore fields"?

Reply:
some of these fields may be chart specific!